### ISSUE TITLE
difference between watchUser and watchUsers

### ISSUE THREAD

**Issue by kevin4dhd** [flutterdata/flutter_data_todos#3]
I'm using this project as a guide, but it happens that watchUser returns DataState, and watchUsers returns a DataStateNotifier, and the worst thing is that in version 0.5.20 it returns DataStateNotifier in both cases, so it makes me really confusing, update the project to the latest version, I know it is used with flutter_data_state, but even so in this example there is no use, so my question is if flutter_data_state should be used with the extra packages that have been implemented, I hope the update of this project example to guide me, thank you.

**Comment by frank06** [comment]
Yes, I'm aware that there are errors. Sorry for that. I will update a few things to make it work with `0.5.20`. I hope this week, but I am not sure if I'll have the time.

**Comment by cskiwi** [comment]
@frank06 any updates on this?

**Comment by frank06** [comment]
Still couldn't get to it. This week I started cleaning up some rough edges in FD and once I get 0.5.21 out I'll fix the sample app. Docs next. Thanks for your patience!

**Comment by frank06** [comment]
This sample app has been revamped, too. It's the same TO-DO app from the Riverpod examples
